src/Components/THIRD-PARTY-NOTICES.txt

```diff
@@ -22,7 +22,7 @@ The actual third-party libraries or other resources may vary depending on the Bl
 2.	AngleSharp  (https://github.com/AngleSharp/AngleSharp)
 3.	SimpleJson (https://github.com/facebook-csharp-sdk/simple-json)
```

Comment:
I'm not sure if we still use it. @SteveSandersonMS, @javiercn , @MackinnonBuck do you know?